queue_job_chunk/components/processor.py

```diff
@@ -0,0 +1,11 @@
+#  Copyright (c) Akretion 2020
+#  License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+
+from odoo.addons.component.core import AbstractComponent
+
+
+# DISCUSSION: abstractcomponent?
+class Processor(AbstractComponent):
+    _name = "processor"
+    _inherit = "base"
+    _collection = "collection.base"
```

Comment:
```suggestion
    _collection = "queue.job.chunk"
```

Reply:
contrary to what the Component documentation says, it seems the collection must be a record when we do:

with queue_job_collection.work_on('res.partner') as work:

if queue_job_collection is not a record, an error is raised.

Because queue.job.chunk has required fields, I would need to create a dummy record with dummy values, which seems complicated